_episodes/10-archeology.md

```diff
@@ -329,30 +82,40 @@ $ git checkout master            # after you are done switch back to "master"
 $ git branch -d museum
 ```
 
-Branches help us to keep the repository organized.
-
----
 
 > ## Archaeology exercise
 >
-> There used to be another test function in the git-archaeology-exercise repository,
-> but now it's gone. You remember that the function started with "test".
-> - Try to find the commits in which the function was introduced and later removed
-> - Inspect the commits. What was the function called?
-> - Create a new branch to point at the commit **before** the commit where that test
->   function was removed, and inspect the `source` folder.
->   Hint: To refer to a commit before a given commit hash, use `<HASH>~1`.
-{: .task}
+> Let us explore the value of these commands in an exercise. We recommend that you
+> do this exercise in groups of two and discuss with your neighbors.
+>
+> 1. Clone this repository:
+>    [https://github.com/tidyverse/rvest](https://github.com/tidyverse/rvest).
```

Comment:
Excellent exercise!

Reply:
Participants seem to have enjoyed it. Step two needs more hints but there is an issue for that.